recipes/dacite/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "dacite" %}
+{% set version = "1.5.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 764c96e0304cb189628686689a163a6a3a8ce7bf3465f0a2d882a8b42f88108f
+
+build:
+  noarch: python
+  number: 0
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - dataclasses  # [py==36]
+  run:
+    - python
+    - dataclasses  # [py==36]
+
+test:
+  imports:
+    - dacite
```

Can we add ``pip check`` here please?
```suggestion
    - dacite
  requires:
    - pip
  commands:
    - pip check
```

---

Cool - will do